freqtrade/exchange.py

```diff
@@ -24,22 +29,32 @@ def init(config: dict) -> None:
     :param config: config to use
     :return: None
     """
-    global _API, EXCHANGE
+    global _CONF, EXCHANGE
 
     _CONF.update(config)
 
     if config['dry_run']:
         logger.info('Instance is running with dry_run enabled')
 
-    use_bittrex = config.get('bittrex', {}).get('enabled', False)
-    if use_bittrex:
-        EXCHANGE = Exchange.BITTREX
-        _API = Bittrex(api_key=config['bittrex']['key'], api_secret=config['bittrex']['secret'])
-    else:
-        raise RuntimeError('No exchange specified. Aborting!')
+    exchange_config = config['exchange']
+    name = exchange_config['name']
+
+    # Find matching class for the given exchange name
+    exchange_class = None
+    for exchange in Exchanges:
+        if name.upper() == exchange.name:
+            exchange_class = exchange.value
+            break
+    if not exchange_class:
+        raise RuntimeError('Exchange {} is not supported'.format(name))
+
+    if not exchange_config.get('enabled', False):
```

Comment:
If we drop the `enabled` property this can be removed

Reply:
Done